src/defaultsettings.cpp

```diff
@@ -76,11 +76,19 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("tooltip_show_delay", "400");
 
 	// Some (temporary) keys for debugging
+	#ifdef NDEBUG
```

Comment:
`DEBUG` isn't defined anywhere, if you want to have block run in release use `#ifndef NDEBUG`.